pkg/plugins/manager/signature/manifest_test.go

```diff
@@ -128,9 +129,9 @@ func TestCalculate(t *testing.T) {
 			{
 				appURL: "https://dev.grafana.com",
 				expectedSignature: plugins.Signature{
-					Status:     plugins.SignatureValid,
-					Type:       plugins.GrafanaSignature,
-					SigningOrg: "Grafana Labs",
+					Status:     plugins.SignatureInvalid,
+					Type:       "",
+					SigningOrg: "",
```

Comment:
I am using a dummy cert for this (to avoid exporting the real public key), so the cert is invalid.

Reply:
I am not 100% sure about this. Shouldn't we test that the plugin shows as signed if the App URL of the instance matches the one in the manifest, and if not it is treated as unsigned? Otherwise I don't think this test case is useful.

The plugin is signed using the "hardcoded fallback" key, so it would just be a matter of replacing the "publicKey" passed to `Calculate` in this test case with the hardcoded fallback one to make the test pass like before, or am I missing something? 🤔